src/Types.jl

```diff
@@ -6,12 +6,10 @@ using UUIDs
 using Random
 using Dates
 import LibGit2
-import REPL
 import Base.string
-using REPL.TerminalMenus
```

Comment:
This probably breaks the `RadioMenu` stuff in this file. However, I think that is more or less useless and should be removed.

Reply:
Is this the only section that is affected?

https://github.com/JuliaLang/Pkg.jl/blob/989b507d26b4f6fd7f28043a772c22e3d5bcfa4b/src/Types.jl#L1076C18-L1084

We could probably create some kind of hook to move that into the extension if we still need it. Perhaps the more pressing thing is developing a test to cover that part of the code.

Reply:
Addressed in [d9138bd](https://github.com/JuliaLang/Pkg.jl/pull/3724/commits/d9138bd07abf110eb61bd807339c84208a30d12a)